<ui version="4.0" >
 <class>SyncDialogUI</class>
 <widget class="QDialog" name="SyncDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Submit Sync</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Job Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mJobNameEdit" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Source</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>1</number>
        </property>
        <item>
         <widget class="QRadioButton" name="mSourceSMO" >
          <property name="text" >
           <string>SMO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mSourceVCO" >
          <property name="text" >
           <string>VCO</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mSourceBH" >
          <property name="text" >
           <string>BH</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="title" >
        <string>Destination</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>1</number>
        </property>
        <item>
         <widget class="QRadioButton" name="mDestSMO" >
          <property name="text" >
           <string>SMO</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mDestVCO" >
          <property name="text" >
           <string>VCO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mDestBH" >
          <property name="text" >
           <string>BAH</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mPathEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="mAddButton" >
       <property name="text" >
        <string>Add Files</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="mFilesTree" >
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>File Path</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Size</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Options</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Priority:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mPrioritySpin" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ProjectCombo" name="mProjectCombo" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Sync Direction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox" >
          <item>
           <property name="text" >
            <string>pull</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>push</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="mDeleteOnCompleteCheck" >
        <property name="text" >
         <string>Delete Job On Completion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Notifications</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Completion:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Errors:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="mEmailCompletionCheck" >
        <property name="text" >
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="mEmailErrorsCheck" >
        <property name="text" >
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="mJabberCompletionCheck" >
        <property name="text" >
         <string>Jabber</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="mJabberErrorsCheck" >
        <property name="text" >
         <string>Jabber</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Total Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mSizeLabel" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOkButton" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectCombo</class>
   <extends>QComboBox</extends>
   <header>blur/Classesui.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mJobNameEdit</tabstop>
  <tabstop>mPrioritySpin</tabstop>
  <tabstop>mEmailErrorsCheck</tabstop>
  <tabstop>mJabberErrorsCheck</tabstop>
  <tabstop>mEmailCompletionCheck</tabstop>
  <tabstop>mJabberCompletionCheck</tabstop>
  <tabstop>mOkButton</tabstop>
  <tabstop>mCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mOkButton</sender>
   <signal>clicked()</signal>
   <receiver>SyncDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>192</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SyncDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>310</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel" >
     <x>312</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
